rclpy/src/rclpy/utils.cpp

```diff
@@ -89,4 +97,103 @@ convert_to_py(void * message, py::object pyclass)
   return py::reinterpret_steal<py::object>(convert(message));
 }
 
+const char *
+get_rmw_implementation_identifier()
+{
+  return rmw_get_implementation_identifier();
+}
+
+void
+assert_liveliness(py::object pyentity)
+{
+  if (PyCapsule_IsValid(pyentity.ptr(), "rclpy_publisher_t")) {
```

Comment:
@gbalke nit: you may use a `py::capsule` e.g.:
```suggestion
assert_liveliness(py::capsule pyentity)
{
  if (0 == strcmp("rclpy_publisher_t", pyentity.name())) {
```